### ISSUE TITLE
cannot click on import button

### ISSUE THREAD

**Issue by mkudlej** [Tendrl/ui#536]
User cannot click on import button because it is covered, see screenshot.
In this case I've used 1024x768 but design should be responsive.

![bug_import_button](https://user-images.githubusercontent.com/5920442/29813492-866c7d24-8caa-11e7-97d5-55714493fae1.png)

Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0

etcd-3.1.9-1.el7.x86_64
glusterfs-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-api-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-cli-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-client-xlators-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-fuse-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-libs-4.0dev-0.85.gitea48cae.el7.centos.x86_64
glusterfs-server-4.0dev-0.85.gitea48cae.el7.centos.x86_64
python-etcd-0.4.5-1.noarch
rubygem-etcd-0.3.0-1.el7.centos.noarch
tendrl-api-1.5.1-1.el7.centos.noarch
tendrl-api-httpd-1.5.1-1.el7.centos.noarch
tendrl-commons-1.5.1-1.el7.centos.noarch
tendrl-monitoring-integration-1.5.1-1.el7.centos.noarch
tendrl-node-agent-1.5.1-1.el7.centos.noarch
tendrl-ui-1.5.1-1.el7.centos.noarch

**Comment by gnehapk** [comment]
@mkudlej Do you see this issue on chrome browser as well? On chrome browser, its working fine for me. 
![screenshot from 2017-08-29 15-42-02](https://user-images.githubusercontent.com/5517376/29816223-f391c01e-8cd0-11e7-969c-1113007c1728.png)

**Comment by mkudlej** [comment]
@gnehapk I don't see it in
```
Chromium Version 59.0.3071.104 (Developer Build) Fedora Project (64-bit)
```
but Firefox is default browser in CentOS and second most used browser on desktop so I think Tendrl should fully support it.

**Comment by gnehapk** [comment]
@mkudlej  Fixed with tendrl-ui-1.5.2-20170925T180704.770936f.noarch.rpm. Please verify.

**Comment by mkudlej** [comment]
I don't see this issue with
```
Mozilla/5.0 (X11; Linux x86_64; rv:55.0) Gecko/20100101 Firefox/55.0
tendrl-ui-1.5.2-20170921T165116.b85723e.noarch
```
